docs/editors/settings.md

```diff
@@ -932,6 +932,12 @@ Whether to enable the Ruff extension. Modifying this setting requires restarting
 
 ### `format.args`
 
+!!! warning "Deprecated"
+
+    This setting is only used by [`ruff-lsp`](https://github.com/astral-sh/ruff-lsp) which is
+    deprecated in favor of the native language server. Refer to the [migration
+    guide](migration.md) for more information.
+
```

Comment:
What do you think of moving all deprecated settings to their own section? E.g. Deprecated `ruff-lsp` Settings`

Reply:
That's an interesting idea, I quickly tried it although I think I'd prefer the current version as one might miss the deprecation warning if they're looking at the settings that requires scrolling past the warning.

<img width="1535" alt="Screenshot 2025-02-04 at 4 02 30 PM" src="https://github.com/user-attachments/assets/39a01db7-4208-4397-90ac-7f876b17ee71" />